### PR TITLE
update to kubernetes-1.8.2

### PR DESCRIPTION
**Pull request by kopiczko**
Towards https://github.com/giantswarm/giantswarm/issues/1554

- remove TPR related code
- vendor dependencies

~~I still need to fix informer integration test.~~ Done.

Follow up PR replacing glide with dep: #131

### CODE REVIEW

**Comment by kopiczko** [comment]
I need that in azure-operator. I can use a branch for the time being.

File: informer/informer_integration_test.go [hunk 1]
```diff
@@ -76,6 +84,19 @@ type TestList struct {
...
+		itemsCopy[i] = item
+	}
+
+	return &TestList{
+		TypeMeta: t.TypeMeta,
+		ListMeta: *t.ListMeta.DeepCopy(),
+		Items:    itemsCopy,
+	}
+}
+
```
Comment (kopiczko) [inline informer/informer_integration_test.go]: This is ugly. I didn't want to bother with that too much.

**Comment by kopiczko** [comment]
I pinned our current operators using operatorkit:

- aws-operator https://github.com/giantswarm/aws-operator/pull/571
- cert-operator https://github.com/giantswarm/cert-operator/pull/130
- endpoint-operator https://github.com/giantswarm/endpoint-operator/pull/22
- flannel-operator https://github.com/giantswarm/flannel-operator/pull/86
- ingress-operator https://github.com/giantswarm/ingress-operator/pull/59
- kvm-operator https://github.com/giantswarm/kvm-operator/pull/304

I left draughtsman-operator (which seems abandoned) and azure-operator (which I'll take care).

@xh3b4sd PTALA I think we can proceed with that.

**Comment by xh3b4sd** [comment]
Please sort draughtsman-operator then. It is not abandoned. I am super big on using it but we do not have a priority on it. When you notice you need to invest too much work into the vendor magic let me know and go ahead. I would like to keep it in sync though.

**Comment by xh3b4sd** [comment]
Also note that the *tpr projects would need to be pinned as well. They also refer to operatorkit and it is likely that glide fucks up.

**Comment by kopiczko** [comment]
PTALA I updated the comment above.

**Comment by kopiczko** [comment]
🎉

**Pull request merged**